backend/src/middleware/filterBubble/filterBubble.spec.js

```diff
@@ -26,24 +32,42 @@ const randomAuthorParams = {
 const categoryIds = ['cat9']
 
 beforeEach(async () => {
-  await Promise.all([
+  const [currentUser, followedAuthor, randomAuthor] = await Promise.all([
     factory.create('User', currentUserParams),
     factory.create('User', followedAuthorParams),
     factory.create('User', randomAuthorParams),
   ])
-  await instance.create('Category', {
+  user = currentUser
+  await neode.create('Category', {
     id: 'cat9',
     name: 'Democracy & Politics',
     icon: 'university',
   })
-  const [asYourself, asFollowedUser, asSomeoneElse] = await Promise.all([
-    Factory().authenticateAs(currentUserParams),
-    Factory().authenticateAs(followedAuthorParams),
-    Factory().authenticateAs(randomAuthorParams),
-  ])
-  await asYourself.follow({ id: 'u2', type: 'User' })
-  await asFollowedUser.create('Post', { title: 'This is the post of a followed user', categoryIds })
-  await asSomeoneElse.create('Post', { title: 'This is some random post', categoryIds })
+  await currentUser.relateTo(followedAuthor, 'following')
+  await factory.create('Post', {
```

Comment:
Beause we don't have https://github.com/thoughtbot/factory_bot :cry: Our custom factories are just a bad copy of this really sophisticated framework, delivering quality since 2008. If we use `neode` for test setup, it got there before there was a corresponding `factory` for that resource. Without our own factories, however, we might setup a state in the database that wouldn't exist, e.g.posts without categories. I prefer to use factories but I don't really mind if there are traces of `neode` in the test setup.

Reply:
makes sense, I saw some examples where you create a user where you are not concerned in the test that it has all the necessary attributes, just that it has an id